### PR TITLE
Rate Component

### PR DESCRIPTION
**Pull request by mxdi9i7**

### CODE REVIEW

File: src/components/Rate/index.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
```
Comment (ssysm) [inline src/components/Rate/index.tsx:1]: `useEffect` imported but never used.

**Comment by ssysm** [comment]
Why use `shortid`? For-loop index not working as expected?

**Comment by mxdi9i7** [comment]
> Why use `shortid`? For-loop index not working as expected?

@ssysm it's not the best practice of React to use index based keys

**Comment by ssysm** [comment]
ok, understood.

**Pull request merged**